src/main/java/com/microsoft/lst_bench/util/FileParser.java

```diff
@@ -103,11 +124,80 @@ public static Map<String, Object> getParameterValues(String parameterValuesFile,
     return values;
   }
 
+  /**
+   * Reads the YAML file and replaces all environment variables (if present). Validates the YAML
+   * file according to the schema. Creates and returns a `TaskLibrary` object.
+   */
+  public static TaskLibrary loadTaskLibrary(String filePath) throws IOException {
+    return createObject(filePath, TaskLibrary.class, true, SCHEMAS_PATH + "task_library.json");
+  }
+
+  /**
+   * Reads the YAML file and replaces all environment variables (if present). Validates the YAML
+   * file according to the schema. Creates and returns a `Workload` object.
+   */
+  public static Workload loadWorkload(String filePath) throws IOException {
+    return createObject(filePath, Workload.class, true, SCHEMAS_PATH + "workload.json");
+  }
+
+  /**
+   * Reads the YAML file and replaces all environment variables (if present). Validates the YAML
+   * file according to the schema. Creates and returns a `ConnectionsConfig` object.
+   */
+  public static ConnectionsConfig loadConnectionsConfig(String filePath) throws IOException {
+    return createObject(
+        filePath, ConnectionsConfig.class, true, SCHEMAS_PATH + "connections_config.json");
+  }
+
+  /**
+   * Reads the YAML file and replaces all environment variables (if present). Validates the YAML
+   * file according to the schema. Creates and returns a `ExperimentConfig` object.
+   */
+  public static ExperimentConfig loadExperimentConfig(String filePath) throws IOException {
+    return createObject(
+        filePath, ExperimentConfig.class, true, SCHEMAS_PATH + "experiment_config.json");
+  }
+
+  /**
+   * Reads the YAML file and replaces all environment variables (if present). Validates the YAML
+   * file according to the schema. Creates and returns a `TelemetryConfig` object.
+   */
+  public static TelemetryConfig loadTelemetryConfig(String filePath) throws IOException {
+    return createObject(
+        filePath, TelemetryConfig.class, true, SCHEMAS_PATH + "telemetry_config.json");
+  }
+
   /**
    * Reads the YAML file and replaces all environment variables (if present). Creates and returns an
    * object of `objectType` class.
    */
-  public static <T> T createObject(String filePath, Class<T> objectType) throws IOException {
-    return yamlMapper.readValue(StringUtils.replaceEnvVars(new File(filePath)), objectType);
+  private static <T> T createObject(String filePath, Class<T> objectType) throws IOException {
```

Comment:
Do you want to leave this as an option or should we just remove it and insist on verification?

Reply:
You are right, currently it was not used anymore, so I simply removed it. If we want to parse JSON without validation for any reason, it should be easy enough to add it.